schemas/service.yml

```
@@ -105,3 +105,13 @@
 
         This id normally changes across restarts, but `service.id` does not.
       example: 8a4f500f
+
+    - name: address
+      level: extended
+      type: keyword
+      short: Address of this service.
+      description: > 
+        Address where this service is hosted.
+
+        This could be a URI, network address (ipv4:port or [ipv6]:port) or a resource path (sockets).
```

```suggestion
        This should be a URI, network address (ipv4:port or [ipv6]:port) or a resource path (sockets).
```

I'd go with "should" since I'm a fan of the definitions in https://datatracker.ietf.org/doc/html/rfc2119#section-3.

---

++